File: src/main/java/com/github/sorusclient/client/adapter/IPlayerEntity.java
/*
 * Copyright (c) 2022. Danterus
 * Copyright (c) 2022. Sorus Contributors
 *
 * SPDX-License-Identifier: MPL-2.0
 */

package com.github.sorusclient.client.adapter;

public interface IPlayerEntity extends ILivingEntity {
    double getHunger();
    double getArmorProtection();
    int getExperienceLevel();
    double getExperiencePercent();
    IPlayerInventory getInventory();
}

File: src/main/kotlin/com/github/sorusclient/client/adapter/IPlayerEntity.kt
/*
 * Copyright (c) 2022. Danterus
 * Copyright (c) 2022. Sorus Contributors
 *
 * SPDX-License-Identifier: MPL-2.0
 */

package com.github.sorusclient.client.adapter

interface IPlayerEntity : ILivingEntity {
    val hunger: Double
    val armorProtection: Double
    val experienceLevel: Int
    val experiencePercentUntilNextLevel: Double
    val inventory: IPlayerInventory
}